src/ray/protobuf/common.proto

```diff
@@ -138,7 +138,11 @@ message RayException {
 /// The runtime environment describes all the runtime packages needed to
 /// run some task or actor.
 message RuntimeEnv {
-  repeated string uris = 1;
+  /// The hashed digest of the runtime env.
+  /// It's supposed to be the unique id for a given runtime env
+  string digest_id = 1;
```

Comment:
I put a specific `digest_id` here in case the id can't be generated by URIs later in the future.

Reply:
Can we specify (in the comment) that this is the md5 of the uploaded uris? (as opposed to the serialized runtime env or something)